modules/conflict_resolver/php/NDB_Menu_Filter_resolved_conflicts.class.inc

```diff
@@ -57,31 +57,30 @@ class NDB_Menu_Filter_Resolved_Conflicts extends NDB_Menu_Filter
         }
 
         $this->columns = array(
-                          'conflicts_resolved.TableName',
-                          'session.CandID',
-                          'candidate.PSCID',
-                          'session.Visit_label',
-                          'conflicts_resolved.ResolutionTimestamp',
+                          'conflicts_resolved.TableName as Instrument',
+                          'session.CandID as CandID',
+                          'candidate.PSCID as PSCID',
+                          'session.Visit_label as Visit_Label',
+                          'conflicts_resolved.ResolutionTimestamp as Timestamp',
                          );
 
         if ($useProjects) {
-            $this->columns[] = 'candidate.ProjectID as Project';
+            $this->columns[] = 'Project.Name as Project';
         }
 
         $this->columns = array_merge(
             $this->columns,
             array(
-             'conflicts_resolved.FieldName',
-             'conflicts_resolved.NewValue',
-             'conflicts_resolved.OldValue1',
-             'conflicts_resolved.OldValue2',
```

Comment:
What happened to NewValue, OldValue2 and CenterID columns? I see it being used in some cases resulting in missing data in the new react table